inference_perf/utils/custom_tokenizer.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from transformers import AutoTokenizer
+
+class CustomTokenizer:
+    def __init__(self, tokenizer_id: str, trust_remote_code: bool = False):
```

Comment:
rename `tokenizer_id` to `pretrained_model_name_or_path`. I suggest we refer to transformers [docs](https://huggingface.co/docs/transformers/v4.50.0/en/model_doc/auto#transformers.AutoTokenizer.from_pretrained) as-is here